test/CMakeLists.txt

```diff
@@ -1,25 +1,5 @@
 foreach(_name
-    benzene/integration
-    benzene/bond-integration
-    benzene/vectors
-    benzene/2d
-    benzene/3d
-    benzene/keyword-magnet
-    benzene/3d-keyword-magnet
-    benzene/2d-keyword-magnet
-    benzene/keyword-spacing
-    benzene/keyword-rotation
-    benzene/keyword-rotation_origin
-    benzene/keyword-radius
-    benzene/int-grid-bond-even
-    benzene/int-cdens
-    benzene/diamag-off
-    benzene/paramag-off
-    benzene/giao-test   
-    c4h4/integration
-    c4h4/read-grid
-    open-shell/3d
-    open-shell/integration
+    benzene/valgrind-integral
```

Comment:
But here all the other tests disappeared.

Reply:
Yes, because I don't know yet how to split the tests into two parts, so that the other tests are executed with the release binary and those with the debug binary. I was thinking of something like  exporting TRAVIS_TEST=2 to differentiate between the two.

Reply:
OK but if this is work in progress then please add "WIP" to your title or make it clear somewhere. Otherwise this could have gotten accidentally merged. I would probably add a CMake option to differentiate.